AsstIntegration

```diff
@@ -189,22 +213,15 @@ script_Main() {
         CURRENT_SCRIPT_STATE="none"
```

Comment:
This can be handled more elegantly. Adding a localized variable for the main function would be more suitable then  setting to "not" in every single case.